app/pages/health/live/getLiveStatus.js

```diff
@@ -0,0 +1,5 @@
+import { status } from '../status';
```

Comment:
you can add this function and getReadyStatus to the same file and they can just live in the health folder

Reply:
I think it's nice to have them separated into a sub-directory that matches their /:route name